src/content/lodash/string/split/lodash.js

```diff
@@ -0,0 +1,9 @@
+import { split } from 'lodash'
+
+const all = split('a-b-c', '-')
+const justTwo = split('a-b-c', '-', 2)
+
+module.exports = {
```

Comment:
please use the format `exports.all = split('a-b-c', '-')` like the one in [this file](https://github.com/cedmax/youmightnotneed/blob/d891c27995f27f1b5e67abe2304a740b908d28ac/src/content/lodash/number/inRange/lodash.js#L3): I did a few tweaks to the UI to show the code nicely when exporting multiple things this way: https://youmightnotneed.com/lodash/#inRange